File: Payhub - SDK/ruby_SDK/PayHubSDK/lib/PayHubSDK/com/payhub/ws/vt/validated_devices.rb
class ValidatedDevices
  include JsonSerializer
  ATTRS=[:enforce_device_validation,:devices,:errors]
  attr_accessor *ATTRS
  VALIDATED_DEVICES_LINK="adminSettings/validatedDevices"
  def initialize
    @devices=Array.new
  end
end
class Devices
  include JsonSerializer
  ATTRS=[:product,:nick_name,:platform,:details,:date_added,:device_id]
  attr_accessor *ATTRS
  def initialize
  end
end

File: Payhub - SDK/c#_SDK/PayHubSDK/PayHubSDK/com/payhub/ws/vt/ValidatedDevices.cs
using PayHubSDK.com.payhub.ws.util;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Runtime.Serialization;
using PayHubSDK.com.payhub.ws.api;

namespace PayHubSDK.com.payhub.ws.vt
{
    [DataContract]
    public class ValidatedDevices
    {
        public static string VALIDATED_DEVICES_LINK = "adminSettings/validatedDevices";

        [DataMember]
        public Boolean enforce_device_validation;
        [DataMember]
        public List<Devices> devices;
        [DataMember]
        public List<Errors> errors;

        public ValidatedDevices() { }

        [DataContract]
        public class Devices
        {
            [DataMember]
            public string product;
            [DataMember]
            public string nick_name;
            [DataMember]
            public string platform;
            [DataMember]
            public string details;
            [DataMember]
            public string date_added;
            [DataMember]
            public long device_id;
            public Devices() { }
        }
    }
    
}
